src/components/header/index.js

```diff
@@ -46,24 +46,18 @@ export default function Header() {
 							aria-label="Browse the code on GitHub"
 							href="https://github.com/preactjs/preact"
 						>
-							<img
-								src="/assets/github.svg"
-								alt="GitHub"
-								width="34"
-								height="33"
-							/>
+							<svg aria-hidden viewBox="0 0 24 24">
```

Comment:
_I think_ we're meant to use aria-hidden here, as it's just content of the link and not relevant to screen readers?